gunagala/config.py

```diff
@@ -76,6 +75,9 @@ def load_config(config_files=None, simulator=None, parse=True, ignore_local=Fals
         # Load local version of config
         if not ignore_local:
             local_version = os.path.join(config_dir, f.replace('.', '_local.'))
+            if not local_version.startswith('/'):
+                local_version = get_pkg_data_filename(local_version)
+
```

Comment:
I think you want to fix the `config_dir` on line 59/60. I see that you are mostly copying what is in 66/67 though.

But seems like you could do:

```python
config_dir = os.path.join(os.path.dirname(__file__), 'data')
```

and then join items to the `config_dir`.